src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+
+  width: $card-width;
+  margin-bottom: 46px;
+  margin-right: 48px;
+  padding: 32px 0 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__icon {
+    width: 160px;
+    height: 134px;
+
+    margin: 0 19px 36px;
+  }
+
+  &__info {
+    display: flex;
+    flex-direction: column;
+
+    margin: 0 16px;
+  }
+
+  &__info-title {
+    margin-bottom: 4px;
+
+    font-size: $medium-size-text;
+    font-weight: $normal-weight;
+    line-height: 18px;
+    color: $emphasized-color;
+  }
+
+  &__info-productCode {
+    margin-bottom: 16px;
+
+    font-size: $small-size-text;
+    font-weight: $light-weight;
+    line-height: 14px;
+    color: $lightened-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+
+    line-height: 14px;
+  }
+
+  &__review-mark {
+    font-size: $small-size-text;
+    font-weight: $light-weight;
+    color: $emphasized-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: $medium-size-text;
+    font-weight: $light-weight;
+    color: $lightened-color;
+  }
+
+  &__price-sum {
+    font-size: 16px;
+    font-weight: $bold-weight;
+    line-height: 18px;
+    color: $emphasized-color;
+  }
+
+  &__button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    box-sizing: border-box;
+    width: 166px;
+    height: 40px;
```

Comment:
Don't hardcode height, set needed paddings, and let content dictate container height